source/docs/essentials/installation.md

```diff
@@ -117,9 +126,16 @@ See [our caching section](/docs/advanced/graphql/dataloaders-and-cache-invalidat
 
 Launch your Front-Commerce project in development mode by running:
 
-```sh
-npm run start
-```
+- For node < 17
+
+  ```sh
+  npm run start
+  ```
+
+- For node >= 17
+  ```sh
+  NODE_OPTIONS=--oenssl-legacy-provider npm run start
```

Comment:
**suggestion:** can't we change the `start` script to force this value?\
I.e: the `bin/front-commerce` script is used to start the app. Can't it do it itself depending on the node version?

Typo
```suggestion
  NODE_OPTIONS=--openssl-legacy-provider npm run start
```

Reply:
If we don't find a solution, we could add a blockquote explaining that it currently is needed because FC is still using webpack 4, and redirecting to https://github.com/webpack/webpack/issues/13572#issuecomment-1011497486 for more details.

Reply:
@mo-fc The `processs` hould include the node version so we could just do this in the bin right?

This is from the bin/front-commerce when running in debugger you can see all the available things on `process`

```bash
// process.versions
ares:'1.18.1'
brotli:'1.0.9'
cldr:'40.0'
icu:'70.1'
llhttp:'6.0.4'
modules:'93'
napi:'8'
nghttp2:'1.45.1'
nghttp3:'0.1.0-DEV'
ngtcp2:'0.1.0-DEV'
node:'16.14.2'
openssl:'1.1.1n+quic'
tz:'2021a3'
unicode:'14.0'
uv:'1.43.0'
v8:'9.4.146.24-node.20'
zlib:'1.2.11'
```

![Code - front-commerce — front-commerce 2022-05-06 at 10 37 25](https://user-images.githubusercontent.com/39598117/167097447-84a83b34-016b-432f-aa72-c3dc460154e8.png)

Reply:
I think we shouldnt mention we are still using webpack 4 on our installation guide. its like a bad omen